lib/tasks/redshift.rake

```diff
@@ -18,6 +18,8 @@ namespace :redshift do
       conn.exec(
         "CREATE TABLE IF NOT EXISTS #{Redshift::SCHEMA_TABLE} ( \
           bearden_id integer, \
+          city character varying, \
+          country character varying, \
```

Comment:
Awesome - I would have totally forgotten this! Does this change mean that we'll have to re-create the databases on Redshift? When should that occur?